docs/faq.rst

```diff
@@ -240,6 +240,7 @@ There are a number of possibilities:
   <https://musicbrainz.org/>`__. You can search on their site to make sure it's
   cataloged there. (If not, anyone can edit MusicBrainz---so consider adding the
   data yourself.)
+- If you are using plugins, make sure that musicbrainz is in the plugin list.
```

Comment:
Could we move this up top the top of the list and reformulate a bit?

Something like this would be superb:
> Make sure you have at least one autotagger extension/plugin enabled. See [here](https://docs.beets.io/en/latest/plugins/index.html#autotagger-extensions) for a list of valid plugins.